packages/frontend/src/main/dialogs/NewSpeckle.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <v-dialog v-if="fe2MessagingEnabled" v-model="dialog" persistent max-width="850px">
```

Comment:
You should not add v-ifs to top level elements, this will result in the component still being mounted & active, just not rendered (a waste of resources) plus it might actually break things occasionally.

Ideally you want to add the v-if to DialogsNewSpeckle wherever you're using it